modules/common/istio/istio-values.tpl

```diff
@@ -170,10 +170,10 @@ security:
   resources:
     requests:
       cpu: 20m
-      memory: 128Mi
+      memory: 32Mi
     limits:
-      cpu: 400m
-      memory: 256Mi
+      cpu: 1
```

Comment:
Why are we increasing the limits so much here?  I think we should b focusing on bumping the requests, not the limits.